assets/docs/configuration/transformations/builtin/jq-full-example.hcl

```diff
@@ -0,0 +1,15 @@
+transform {
```

Comment:
These files are displayed directly in the documentation, and serve as our configuration reference docs. For both the full and minimal example, each option needs a comment describing what it is.

The other transformations should serve as an easy example for what that should look like - timeout and snowplow_mode can likely just be copy pasted.